packages/lifecycle/src/index.ts

```diff
@@ -3,6 +3,8 @@ import { safeReadPackageFromDir } from '@pnpm/read-package-json'
 import exists from 'path-exists'
 import runLifecycleHook, { RunLifecycleHookOptions } from './runLifecycleHook'
 import runLifecycleHooksConcurrently, { RunLifecycleHooksConcurrentlyOptions } from './runLifecycleHooksConcurrently'
+// @ts-expect-error
```

Comment:
you can create a new file with the following content to get a type definition:

```ts
declare module 'patch-package/dist/applyPatches' {
    export function applyPatch()
}
```

(Do not include any top-level import/export otherwise this will not work)